app/components/rdf-input-fields/mandataris-fractie-selector.js

```diff
@@ -40,27 +36,48 @@ export default class MandatarisFractieSelector extends InputFieldComponent {
   }
 
   async load() {
-    await this.loadBestuursorgaan();
-    await Promise.all([this.loadProvidedValue(), this.loadFracties()]);
+    await Promise.all([this.loadBestuursorganen(), this.loadProvidedValue()]);
     this.initialized = true;
   }
 
-  async loadBestuursorgaan() {
-    this.bestuursorgaanUris = loadBestuursorgaanUrisFromContext(
-      this.storeOptions
+  async loadBestuursorganen() {
+    this.bestuurseenheid = this.currentSession.group;
+    const queryParams = this.router.currentRoute.queryParams;
+
+    const bestuursorganen = await this.store.query('bestuursorgaan', {
+      'filter[bestuurseenheid][id]': this.bestuurseenheid.id,
+      'filter[:has-no:deactivated-at]': true,
+      'filter[:has-no:is-tijdsspecialisatie-van]': true,
+      'filter[classificatie][id]':
+        this.decretaleOrganen.classificatieIds.join(','), // only organs with a political mandate
```

Comment:
this.decretaleOrganen is undefined: bug?

Reply:
fixed it, forgot to inject the service